kwc-dialog.html

```diff
@@ -70,36 +81,60 @@
             }
 
             .toolbar h2 {
-                margin: auto 15px;
+                margin: auto 15px auto 15px;
                 @apply --layout-flex;
+                color: var(--color-chateau, #414A51);
+                font-size: 20px;
             }
 
             .actions button.primary, .actions button.secondary,
             .actions ::slotted(button.primary), .actions ::slotted(button.secondary) {
                 @apply --kano-button;
-                background: var(--kwc-dialog-secondary-color, #9fa4a8);
-                transition: opacity 200ms ease-in;
+                background-color: var(--kwc-dialog-secondary-color, #9fa4a8);
+                transition: all 200ms ease-in;
+                margin-left: 16px;
+                border-radius: 3px;
             }
 
-            .actions ::slotted(button.primary), .actions ::slotted(button.secondary) {
-                margin-left: 3px;
+            .actions button.primary:first-child, .actions button.secondary:first-child,
+            .actions ::slotted(button.primary:first-child), .actions ::slotted(button.secondary:first-child) {
+                margin-left: 0px;
             }
 
-            .actions button.primary:hover, .actions button.secondary:hover,
-            .actions ::slotted(button.primary:hover), .actions ::slotted(button.secondary:hover) {
-                opacity: 0.8;
+            .actions button.primary:hover,
+            .actions ::slotted(button.primary:hover) {
+                background-color: var(--kwc-dialog-primary-hover-color, #c95924);
+                color: #f0f0f0;
+            }
+
+            .actions button.secondary:hover,
+            .actions ::slotted(button.secondary:hover) {
+                background-color: var(--kwc-dialog-secondary-hover-color, #7f8386);
+                color: #f0f0f0;
             }
 
             .actions button.primary, .actions ::slotted(button.primary) {
                 background-color: var(--kwc-dialog-primary-color, var(--color-kano-orange));
             }
+
+            @media only screen and (max-width: 420px) {
+                h2 {
+                    font-size: 16px;
+                }
+
+                .actions button,
+                .actions ::slotted(button.primary),
+                .actions ::slotted(button.secondary) {
+                    font-size: 12px !important;
+                    padding: 8px 15px !important;
+                }
+            }
         </style>
         <div class="toolbar">
             <h2>{{title}}</h2>
             <div class="actions">
                 <slot name="actions">
-                    <button class="secondary" dialog-dismiss>Cancel</button>
-                    <button class="primary" dialog-confirm>Done</button>
+                    <button class="secondary" dialog-dismiss>Cancel</button><button class="primary" dialog-confirm>Done</button>
```

Comment:
Was the line break causing issues? It was kind of easier to read, though...

Reply:
The problem there was that having these on separate line inserts a whitespace between the buttons which adds about 5px to the margin. This whitespace doesn't get inserted when you slot some buttons in, weirdly, which then creates an inconsistency.

This is annoying but I couldn't find any other way of getting rid of the space.

Reply:
I feel you pain! heheh All good! It has to do with the `display: inline` thing I thing... I had the same issue with `kwc-soft-buttons`